public/data/Mohasin-Haque.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "Mohasin Haque",
+  "bio": "Techie Teacher at Techie Tessie.",
+  "avatar": "https://github.com/Mohasin-Haque.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Mohasin-Haque",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/mohasin_haque",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Linkedin",
+      "url": "https://www.linkedin.com/in/mohasin-haque-759786200/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on Facebook",
+      "url": "https://www.facebook.com/ajad.ali.3766",
+      "icon": "facebook"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started YouTube",
+      "image": "https://github.com/Mohasin-Haque",
+      "date": "August 2021",
+      "icon": "youtube",
+      "color": "red",
+      "description": "Free programming videos every Friday",
+      "url": "https://www.youtube.com/techietessie"
+    }
+  ]
+}
```

Missing an endline. I'll make a PR for it.
```suggestion
}

```